test/e2e/e2ethanos/services.go

```diff
@@ -494,6 +494,12 @@ func NewRulerBuilder(e e2e.Environment, name string) *RulerBuilder {
 	}
 }
 
+// Force no replica label; by default we use replica label in tests.
+func (r *RulerBuilder) WithoutReplicaLabel() *RulerBuilder {
+	r.replicaLabel = ""
+	return r
+}
```

Comment:
Can we use `WithReplicaLabel("")` directly?

Reply:
You're right, that just as readable and simpler, I removed that option.